src/app/file-manager/services/bucket.service.ts

```diff
@@ -9,6 +9,7 @@ import { Type } from '@app/file-manager/models/type';
 export class BucketService {
 
   static delimiter: string;
+  static isResponseComplete = false;
```

Comment:
Why is this `static`? I know we only have a single 'download' view but it doesn't seem correct to make this state static when it could be a field on `BucketService`, and the file explorer component already has a field `bucketService`.